admin-console/4.4/notification-drawer/notification-drawer.md

```diff
@@ -0,0 +1,28 @@
+# Notification Drawer
+
+## Masthead Icon
+![](img/notifications-closed.png)
+- When alerts are firing, a special signifier icon appears in the masthead. Clicking on the icon opens the notification drawer.
+
+## Critical Alerts
+![](img/notifications-open.png)
+- As this drawer is an endopoint for alertmanager, silenced and pending alerts do not appear here.
+- Clicking on an alert takes the user to the details page for the alert.
+- The alertmanager config could potentially be set up to send non-critical alerts here, but by default that would not be the case.
+- The critical alerts group should be expanded by default  and all others should be collapsed.
+
+## Available Updates
+![](img/notifications-updates.png)
+- Updates for the cluster or for specific operators could also be listed here.
+- Clicking on one of these cards would take the user to a page with information about the updates.
+- Action buttons to install the update immediately could also be provided.
+
+## Empty State
+![](img/notifications-empty.png)
+- When there are no critical alerts firing, the `Critical alerts` group should still be present but should contain an empty state pattern.
```

Comment:
Since technically there are Alerts which can be firing or not...should we be very clear with the text used here and say "Critical Alerts Firing"? It's pretty long but I fear people will mismatch their understanding when they compare this to the Alerts list which will list all Alerts whether firing or not.

Reply:
I think it should be pretty clear because non-firing alerts (technically alerting rules) are filtered out by default on the full list.

Reply:
Cool with me if you think it will be clear! I'll close this one :)